server/src/ScoreGuideApp.ts

```diff
@@ -0,0 +1,61 @@
+// tslint:disable-next-line: no-require-imports no-var-requires
+require( 'dotenv' ).config();
+
+import * as path from 'path';
+import * as Express from 'express';
+import * as bodyParser from "body-parser";
+import * as morgan from "morgan";
+import { loggerApi } from "./Logger";
+import { APIRoute } from './routes/API';
+import { ItemParser } from './ItemParser';
+
+
+/**
+ * The ScoreGuideApp encapsulates an Express Application for the Scoring Guide
+ * Server routes, configuration, and initialization is here.
+ * @member {port: string} the port number that the server will listen on
+ * @member {app: Express.Application} the Express.Application
+ * @member {router: APIRouter} routes to the pdf api
+ * @class ScoreGuideServer
+ */
+export class ScoreGuideApp {
+    port: string;
+    app: Express.Application;
+    router: APIRoute;
+
+    constructor () {
+        this.port = process.env.PORT;
```

Comment:
You might want to add something like
this.port = process.env.PORT || 3000 // in case the environment doesn't specify a port number

Reply:
That is on line 56 in the listen() method

Reply:
Do you think we should do it i the constuctor though? I can see an argument for that

Reply:
Cool, the pr looks fine to me then!